modules/govuk/lib/facter/vdc.rb

```diff
@@ -0,0 +1,13 @@
+# Fact which specifies the name of the vDC of a node
+
+Facter.add("vdc") do
+  setcode do
+    env_octet = Facter.value(:ipaddress).split('.')[2].to_i
+    vdc = Facter.value(:domain).split('.').first
+    if env_octet > 7
+      vdc + '_dr'
+    else
+      vdc
+    end
+  end
+end
```

Comment:
It'd be good to add some tests for this Facter logic - we've got a couple of examples like in `fqdn_metrics_spec.rb`. Happy to pair on that if you want.